### DISCUSSION TITLE
Suggestion: World Map Plugin - More detailed tooltips for Lovakengj Minecart Network

### DISCUSSION THREAD

**Discussion by MoneyHalver** [runelite/runelite#17923] [Ideas]
Is there any shot we could get the names of the minecart locations in the tooltips on the world map plugin? I'm sure I'm not the only person that would benefit from this change.

**Comment by kyle-neal** [comment]
@MoneyHalver there's this plugin, which helps: https://www.reddit.com/r/2007scape/comments/17ssccp/introducing_the_teleport_maps_plugin_available_now/.

But I agree, it would be nice to have tool tips on the Lovakengj Minecart Network.  On the world map itself as well

**Comment by cdfisher** [comment]
There's a PR I have open that would do this, #17930
